kakaemma/My-Diary-React
# #162243954 Users should be able to register with My Diary

#### What does this PR do?
This PR adds functionality to register users.

### How should this be manually tested?
clone the repository with `git clone`
create another branch off the current one with `git checkout -b <your_branch_name>` 
run `git pull origin ft-users-should-register`
Run `npm run mydiary` to install all dependencies and start the application
Enter the UR `http://localhost:3000/` in your browser.
click signup and use the signup page to register a user

#### What are the relevant pivotal tracker stories?
[#162243954](https://www.pivotaltracker.com/story/show/162243954)

#### Any background context you want to add?
N/A

## Checklist: I completed these to help reviewers
